std/stdio.d

```diff
@@ -3258,14 +3258,20 @@ struct LockingTextReader
 }
 
 /**
- * Indicates whether $(D T) is a file handle of some kind.
+ * Indicates whether $(D T) is a file handle, i.e. the type
+ * is implicitly convertable to $(LREF File) or a pointer to a
+ * $(REF FILE, core,stdc,stdio).
+ *
+ * Returns:
+ *      `bool`
```

Comment:
A `Returns:` section is needed whenever a return value is there. Even if it is redundant. The reasons are consistency and automated tooling.